src/scheduler/jobs-service.factory.js

```diff
@@ -0,0 +1,102 @@
+(function () {
+    'use strict';
+
+    angular.module('motech-scheduler')
+        .factory('JobsService', jobsService);
+
+    jobsService.$inject = ['$resource', 'ServerService'];
+    function jobsService($resource, ServerService) {
+        var listener = {},
+            currentJob = {},
+            jobs = {},
+            params = {
+                name: "",
+                rows: "10",
+                page: "1",
+                sortColumn: "name",
+                sortDirection: "asc",
+                activity: "NOTSTARTED,ACTIVE,FINISHED",
```

Comment:
If we should remove these default params, or set them in a place where they will be seen by the scheduler.controller's search property...

it would probably make the double page load go away